data/kubernetes-operator-published-branches.yaml

```diff
@@ -39,17 +22,6 @@ git:
       - 'v1.8'
       - 'v1.7'
       - 'v1.6'
-      - 'v1.5'
-      - 'v1.4'
-      - 'v1.3'
-      - 'v1.2'
-      - 'v1.1'
-      - 'v1.0'
-      - 'v0.12'
-      - 'v0.11'
-      - 'v0.10'
-      - 'v0.9'
-      - 'v0.8'
```

Comment:
Not sure we should've removed this part, per my memory of how the old tools worked. But let's see!

Reply:
@melissamahoney-mongodb seems to have worked